client/src/components/SelectionDialog/selectionTypes.ts

```diff
@@ -6,21 +6,19 @@ export const SELECTION_STATES = {
 
 export type SelectionState = (typeof SELECTION_STATES)[keyof typeof SELECTION_STATES];
 
-export interface FieldEntry {
-    key: string;
-    label?: string;
-    sortable?: boolean;
-}
-
 export interface SelectionItem {
     id: string;
     label: string;
-    details: string;
+    time?: Date | string;
+    update_time: Date | string;
+    details?: string;
     isLeaf: boolean;
     url: string;
+    size?: number;
+    variant?: SelectionState;
+    tags?: string[];
     _rowVariant?: SelectionState;
```

Comment:
What is the difference between `variant` and `_rowVariant`? can we use `selectionState?: SelectionState`?